.github/workflows/deploy-documentation.yml

```diff
@@ -0,0 +1,32 @@
+name: Deploy Documentation
+on:
+  push:
+    #branches: [ $default-branch ]
+
+jobs:
+  deploy-documentation:
+    runs-on: ubuntu-16.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Dokka
+        run: |
+          ./gradlew dokkaHtml
+          ls core/build/dokka/html/
+
+      - name: Deploy
+        run: |
+          git config user.email "${{ secrets.CONX_GH_PAGES_USER_EMAIL }}"
+          git config user.name "${{ secrets.CONX_GH_PAGES_USER_NAME }}"
+          git fetch
+          git checkout gh-pages
+          rsync -av --delete core/build/dokka/html/ .
+          git add ./core
+          git add ./images
+          git add navigation.html
+          git add ./scripts
+          git add ./styles
+          git commit -m 'Updating dokka documentation'
+          git push origin gh-pages
```

Comment:
Missing trailing newline.

Reply:
Still messing with stuff. The rsync command david posted is complaining that items are being deleted before getting copied.

Reply:
> complaining that items are being deleted before getting copied.

That makes sense, as your destination includes your source. Not sure of a good way around that, without doing a similar dance we had to do with `mv` anyways. 🤷

Reply:
There are options for that - see the man page for `--delete-during/before/after` etc

Reply:
Yep. Found delete-after in the docs which fixed it, but doesn't really matter now because of the plugin.